biometric/biometric/build.gradle

```diff
@@ -87,7 +87,7 @@ androidx {
     name = "Biometric"
     type = SoftwareType.PUBLISHED_LIBRARY
     inceptionYear = "2018"
-    description = "The Biometric library is a static library that you can add to your Android application. It invokes BiometricPrompt on devices running P and greater, and on older devices will show a compat dialog. Compatible on devices running API 14 or later."
+    description = "The Biometric library is a static library that you can add to your Android application. It invokes BiometricPrompt on devices running P and greater, and on older devices will show a compat dialog. Compatible on devices running API 23 or later."
```

Comment:
We can just remove `Compatible on devices running API 23 or later`. It doesn't add much value.